Common/cpp/Registries/EventHandlerRegistry.cpp

```diff
@@ -6,17 +6,20 @@ namespace reanimated {
 void EventHandlerRegistry::registerEventHandler(
     std::shared_ptr<WorkletEventHandler> eventHandler) {
   const std::lock_guard<std::mutex> lock(instanceMutex);
-  eventMappings[eventHandler->eventName][eventHandler->id] = eventHandler;
-  eventHandlers[eventHandler->id] = eventHandler;
+  const std::string &eventName = eventHandler->getEventName();
```

Comment:
What about using `const auto &` here instead? 🤔 
```suggestion
  const auto &eventName = eventHandler->getEventName();
```